### PR TITLE
Add native implementation of format

### LINKED ISSUE #60: Native Date format using toLocaleDateString
Hi, I really like this! thanks for your effort!

I Know it's not exactly the same, but i think you can add **toLocaleDateString** as an native option for date formatting. More in the link below:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

### PR DESCRIPTION
**Pull request by Amorymeltzer**
Closes #60

### CODE REVIEW

**Comment by Amorymeltzer** [comment]
I think failures on node 8 are related to not having full intl support, see: https://github.com/nodejs/node/issues/8500 and https://github.com/nodejs/node/issues/19214.

Not sure if specifying 'en-US' would fix that.

**Comment by stevemao** [comment]
@cylim Yes good idea :)

File: __tests__/index.js [hunk 1]
```diff
@@ -395,22 +395,37 @@ describe('Manipulate', () => {
...
+    const d = date.format(new Date(time), 'eeee, MMMM d, yyyy, h:mm:ss aa', {
       awareOfUnicodeTokens: true,
     });
-    const day = dayjs(time).format('dddd, MMMM D YYYY, h:mm:ss A');
+    const day = dayjs(time).format('dddd, MMMM D, YYYY, h:mm:ss A');
     const l = DateTime.fromMillis(time).toFormat(
-      'EEEE, MMMM d yyyy, h:mm:ss a'
+      'EEEE, MMMM d, yyyy, h:mm:ss a'
     );
+    expect(m).toBe(n);
```
Comment (ODudek) [inline __tests__/index.js]: This test failed. Native `new Date` does not support am/pm as far as i know :)

Comment (Amorymeltzer) [reply __tests__/index.js]: Passed/passes for me (and in Travis?), but does adding `hour12: true` in the object do it for you?

**Comment by cht8687** [comment]
could you resolve the conflicts? so that we can merge

**Comment by Amorymeltzer** [comment]
@cht8687 Done; apologies for not doing so earlier, I had thought this stalled [waiting for removal of node 8](https://github.com/you-dont-need/You-Dont-Need-Momentjs/pull/83#pullrequestreview-308964541)

**Review by stevemao — APPROVED** [review]

**Comment by Amorymeltzer** [comment]
Conflict comes from #116, which largely (though not equivalently?) supersedes this

**Pull request closed without merging**